pkg/server/server.go

```diff
@@ -66,9 +66,11 @@ func (s *Server) createRouters() *negroni.Negroni {
 
 	n.UseHandler(r)
 
-	r.HandleFunc("/data/{deviceId}", dataController.GetAll).Methods("GET")
-	r.HandleFunc("/data/{deviceId}/sensor/{id}", dataController.GetByID).Methods("GET")
+	r.HandleFunc("/data/{user}", dataController.GetAll).Methods("GET")
```

Comment:
Why did you change this? `deviceId` was correct.

Reply:
Done!